### PR TITLE
[Microsoft Graph Security] update docs

### PR DESCRIPTION
**Pull request by michal-dagan**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

fixes: [link to the issue](https://jira-dc.paloaltonetworks.com/browse/CIAC-9974)

## Description
Documentation and metadata improvements.

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/MicrosoftGraphSecurity/Integrations/MicrosoftGraphSecurity/MicrosoftGraphSecurity.yml [hunk 1]
```diff
@@ -902,7 +902,7 @@ script:
       - 'true'
       - 'false'
       required: false
-    description: Lists edicovery cases.
+    description: Get a list of the edicovery cases.
```
Comment (julieschwartz18) [inline Packs/MicrosoftGraphSecurity/Integrations/MicrosoftGraphSecurity/MicrosoftGraphSecurity.yml:905]: Recommend:
Get a list of the eDiscovery cases.

File: Packs/MicrosoftGraphSecurity/Integrations/MicrosoftGraphSecurity/MicrosoftGraphSecurity_description.md [hunk 2]
```diff
@@ -20,10 +12,14 @@ There are two application authentication methods available:
 
 Depending on the authentication method that you use, the integration parameters might change.
 
-To allow us to access to Microsoft Graph Security, an admin has to approve our app using an admin consent flow, by clicking this [link](https://oproxy.demisto.ninja/ms-graph-security).
+To use the **Cortex XSOAR application** and allow Cortex XSOAR access to Microsoft Graph Security an administrator has to approve our app using an admin consent flow by clicking this **[link](https://oproxy.demisto.ninja/ms-graph-security)**.
```
Comment (julieschwartz18) [inline Packs/MicrosoftGraphSecurity/Integrations/MicrosoftGraphSecurity/MicrosoftGraphSecurity_description.md:15]: Recommend:
To allow Cortex XSOAR access to Microsoft Graph Security, an administrator has to approve the Cortex XSOAR app using an admin consent flow by clicking this [link](https://oproxy.demisto.ninja/ms-graph-security).

File: Packs/MicrosoftGraphSecurity/Integrations/MicrosoftGraphSecurity/MicrosoftGraphSecurity_description.md [hunk 3]
```diff
@@ -20,10 +12,14 @@ There are two application authentication methods available:
...
 After authorizing the Cortex XSOAR app, you will get an ID, Token, and Key which should be inserted in the integration instance settings fields.
 If you previously had an API V1 configured based on the credentials obtained from this method, refer to the link above to gain new credentials with the relevant permissions.
 
+### Important
+1. The *eDiscovery* and *Threat Assessment* commands are only supported when using the `Authorization Code flow` with `Delegated (work or school account)` permission type.
```
Comment (julieschwartz18) [inline Packs/MicrosoftGraphSecurity/Integrations/MicrosoftGraphSecurity/MicrosoftGraphSecurity_description.md:20]: Recommend:
The ***eDiscovery*** and ***Threat Assessment*** commands are only supported when using the `Authorization Code flow` with `Delegated (work or school account)` permission type.

File: Packs/MicrosoftGraphSecurity/Integrations/MicrosoftGraphSecurity/MicrosoftGraphSecurity_description.md [hunk 4]
```diff
@@ -20,10 +12,14 @@ There are two application authentication methods available:
...
+2. When using `Authorization Code flow`, the connection should be tested using the `!msg-auth-test` command.
```
Comment (julieschwartz18) [inline Packs/MicrosoftGraphSecurity/Integrations/MicrosoftGraphSecurity/MicrosoftGraphSecurity_description.md:21]: Recommend:
 When using `Authorization Code flow`, the connection should be tested using the ***!msg-auth-test*** command.

File: Packs/MicrosoftGraphSecurity/Integrations/MicrosoftGraphSecurity/README.md [hunk 5]
```diff
@@ -23,23 +22,23 @@ For more information, see: https://github.com/microsoftgraph/security-api-soluti
...
 3. User.Read.All - Application (Only required if using the deprecated commands: `msg-get-user` and `msg-get-users`)
 
-Alerts v2:
+**Alerts v2**:
 
 1. SecurityAlert.Read.All - Application (required for the commands: `msg-search-alerts` and `msg-get-alert-details`)
 2. SecurityAlert.ReadWrite.All - Application (required for updating alerts with the commands: `msg-update-alert` and `msg-create-alert-comment`)
 
-EDiscovery:
+**EDiscovery**:
```
Comment (julieschwartz18) [inline Packs/MicrosoftGraphSecurity/Integrations/MicrosoftGraphSecurity/README.md:36]: Recommend:
**eDiscovery**

File: Packs/MicrosoftGraphSecurity/Integrations/MicrosoftGraphSecurity/README.md [hunk 6]
```diff
@@ -2168,9 +2150,29 @@ Retrieve all threat assessment requests.
 >| 49c5ef5b-1f65-444a-e6b9-08d772ea2059 | "2019-11-27T03:30:18.6890937Z"| mail | block| spam| pending| administrator | avishaibrandies@microsoft.com |63798129-a62c-4f9e-2c6d-08d772fcfb0e|spam attempt.|notJunk|
 >| ab2ad9b3-2213-4091-ae0c-08d76ddbcacf | 2019-11-20T17:05:06.4088076Z| mail | block| malware| pending| administrator | avishaibrandies@microsoft.com |63798129-a62c-4f9e-2c6d-08d772fcfb0e|Malware attempt.|notJunk|
 
+
+### msg-generate-login-url
+
+***
+Generate the login URL used for authorization code flow.
```
Comment (julieschwartz18) [inline Packs/MicrosoftGraphSecurity/Integrations/MicrosoftGraphSecurity/README.md:2157]: Recommend:
Generate the login URL used for the authorization code flow.

File: Packs/MicrosoftGraphSecurity/Integrations/MicrosoftGraphSecurity/README.md [hunk 7]
```diff
@@ -2168,9 +2150,29 @@ Retrieve all threat assessment requests.
...
+There is no context output for this command.
+
+#### Human Readable Output
 >### Authorization instructions
 >1. Click on the [login URL]() to sign in and grant Cortex XSOAR permissions for your Azure Service Management.
 You will be automatically redirected to a link with the following structure:
 >```REDIRECT_URI?code=AUTH_CODE&session_state=SESSION_STATE```
 >2. Copy the `AUTH_CODE` (without the `code=` prefix, and the `session_state` parameter)
-and paste it in your instance configuration under the **Authorization code** parameter.
+and paste it in your instance configuration under the **Authorization code** parameter.
```
Comment (julieschwartz18) [inline Packs/MicrosoftGraphSecurity/Integrations/MicrosoftGraphSecurity/README.md:2178]: Recommend:
and paste it in the *Authorization code* parameter under your integration instance settings.

**Review by julieschwartz18 — CHANGES_REQUESTED** [review]
Doc review complete. Note: I only reviewed the updates

File: Packs/MicrosoftGraphSecurity/ReleaseNotes/2_2_12.md [hunk 8]
```diff
@@ -0,0 +1,5 @@
+
+#### Integrations
+
+##### Microsoft Graph Security
+Documentation and metadata improvements.
```
Comment (julieschwartz18) [inline Packs/MicrosoftGraphSecurity/ReleaseNotes/2_2_12.md:5]: Is this enough detail? Do you need to mention anything about improved descriptions for eDiscovery and Threat Assessment commands?

**Review by bziser — APPROVED** [review]

**Pull request merged**